integration-tests/README.md

```diff
@@ -3,13 +3,36 @@
 This folder contains integration tests scripts to start firehose ingestion nodes for metachain and shard(0) with a local
 testnet and check indexed data validity for custom a scenario(`ESDTIssue`).
 
+### Data flow
+
+This chapter illustrates how MultiversX handles exported data using WebSocket Exporter Outport Driver.
+
+Consider a network consisting of nodes in a sharded architecture, all being orchestrated by a metachain. Adding an
+observer (in either shard), and making it act as a WebSocket Exporter(by enabling a config file), it will extract and
+export data. This exported data is then ingested by the Firehose Connector, which processes it through the StreamingFast
+framework, enabling efficient data streaming and analysis.
+
+```
++-------------------------+       +------------------------+       +-----------------------+
+|   Local Testnet         | ----> |   WebSocket Exporter   | ----> |   Firehose Connector  |
++-------------------------+       +------------------------+       +-----------------------+
+|   +-------+ +---------+ |       |   +----------------+   |       |   +----------------+  |
+|   |Shard1 | |  Shard2 | |       |   | Observer in    |   |       |   | StreamingFast  |  |
+|   +-------+ +---------+ |       |   | Shard x        |   |       |   | Data Processing|  |
+|   |Shard3 | |Metachain| |       |   +----------------+   |       +-----------------------+
```

Comment:
add `+----------------+` below `Data processing` of the `Firehose Connector` table

Reply:
Right, changed as suggested